vmfloaty.gemspec

```diff
@@ -6,10 +6,10 @@ require 'vmfloaty/version'
 Gem::Specification.new do |s|
   s.name = 'vmfloaty'
   s.version = Vmfloaty::VERSION
-  s.authors = ['Brian Cain']
-  s.email = ['brianccain@gmail.com']
+  s.authors = ['Puppet']
```

Comment:
Hey! I'd prefer to still be the author here since I wrote it :p Feel free to update the email though to include puppet, but I'd rather not erase the fact that I authored this project :smile:

Reply:
I agree, Brian wrote this and we should continue to credit him in the gemspec. Please update this to an array with both Brian and Puppet, @genebean.

Reply:
Will do

Reply:
Thank you @genebean :pray:

Reply:
I’ll push an update as soon as my internet gets unbusted

Reply:
This should be good now